VerilogCodingStyle.md

```diff
@@ -776,6 +776,15 @@ assign state_next = condition_b ?
     state;
 ```
 
+However, the nested ternary can be written in an unambiguous way in many
+cases. Parenthesis are then not required.
```

Comment:
Maybe the other way around, e.g. something like `Parentheses may be omitted if the code formatting conveys the same information.`

And nit: one sentence per line.

Reply:
I like that rewording.

Reply:
Yep. Done.